semgrep/tests/e2e/rules/autofix/csv-writer.yaml

```diff
@@ -0,0 +1,14 @@
+rules:
+- id: python.lang.security.unquoted-csv-writer.unquoted-csv-writer
+  patterns:
+  - pattern-not: csv.writer(..., quoting=csv.QUOTE_ALL, ...)
+  - pattern: csv.writer(...)
+  fix: s/\)/, quoting=csv.QUOTE_ALL)/g
```

Comment:
This probably doesn't play well if there are any close parenthesis in an argument to csv writer. The code will replace the first close parent encountered.

Reply:
OK, yeah. So that means the proper 'fix' is making sure it's at the end of the line or something similar.

I realize this isn't the ideal autofix solution, but it at least makes autofixes attainable for demo purposes -- or for the committed! 😝